app.py

```diff
@@ -1,19 +1,103 @@
 import flask
```

Comment:
The entirety of this file will need to be fixed to merge in with the back end as it stands now as we figure out how to properly route all of these pages